Bottom: 91a4dd86097ddfc02fac15f17c2de92c98fc096d
Top:    2a9beeb34840445cdda54b8c6ac0bcc2f7f829bf
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 14:59:07 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 35c7069..f6cf852 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1295,6 +1295,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
        *
        */
 
+      fprintf (stderr, "Subtracting pin + via drills\n");
+      r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+      r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
       if (info.poly == NULL)
         {
           fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
